pkg/controller/velero/velero.go

```diff
@@ -31,18 +31,35 @@ import (
 const (
 	awsCredsSecretIDKey     = "aws_access_key_id"     // #nosec G101
 	awsCredsSecretAccessKey = "aws_secret_access_key" // #nosec G101
-	veleroImageRegistry     = "docker.io/velero"
-	veleroImageRegistryCN   = "registry.docker-cn.com/velero"
-	veleroImageTag          = "velero:v1.3.1"
-	veleroAwsImageTag       = "velero-plugin-for-aws:v1.0.1"
-	credentialsRequestName  = "velero-iam-credentials"
+
+	veleroImageRegistry   = "docker.io/velero"
+	veleroImageRegistryCN = "registry.docker-cn.com/velero"
+
+	veleroImageTag    = "velero:v1.3.1"
+	veleroAwsImageTag = "velero-plugin-for-aws:v1.0.1"
+	veleroGcpImageTag = "velero-plugin-for-gcp:v1.0.1"
+
+	credentialsRequestName = "velero-iam-credentials"
+)
+
+var (
+	awsChinaRegions = []string{"cn-north-1", "cn-northwest-1"}
 )
 
 func (r *ReconcileVelero) provisionVelero(reqLogger logr.Logger, namespace string, platformStatus *configv1.PlatformStatus, instance *veleroInstallCR.VeleroInstall) (reconcile.Result, error) {
 	var err error
 
-	locationConfig := make(map[string]string)
-	locationConfig["region"] = platformStatus.AWS.Region
+	var locationConfig map[string]string
+	switch platformStatus.Type {
+	case configv1.AWSPlatformType:
+		locationConfig = map[string]string{
+			"region": platformStatus.AWS.Region,
+		}
+	case configv1.GCPPlatformType:
+		// No region configuration needed for GCP
+	default:
+		return reconcile.Result{}, fmt.Errorf("unable to determine platform")
+	}
```

Comment:
A thought maybe for later, but if we added a `Driver` pointer to the `ReconcileVelero` struct and allocate it in `newReconciler()`, then the `provisionVelero()` method would have access to the driver and we could add methods like `GetLocationConfig()` and `CreateCredentialsRequest()` to the driver to avoid platform-based switch statements like this.